### PR TITLE
feature(login): login page should be responsive

### PR DESCRIPTION
**Pull request by abidex4yemi**
# Description

* Tweak login page for responsiveness

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/pages/Login/Login.js [hunk 1]
```diff
@@ -1,77 +1,80 @@
...
 
-function Login({ setUser }) {
-  const [credentials, changeCredentials] = useState({
+function Login({ setUser, ...props}) {
+  const [state, setState] = useState({
     username: '',
-    password: ''
+    password: '',
+    errors: {},
+    loginIn: false
```
Comment (daquinons) [inline src/components/pages/Login/Login.js]: I know that `loginIn` is about saying if it's currently in the process of login in order to display the Loading... indicator. But at the beginning I thought this was about if it was successfully logged in. Maybe just 'loading' would work better and make it easier to see on the first read.

File: src/components/pages/Login/Login.js [hunk 2]
```diff
@@ -1,77 +1,80 @@
...
   });
 
-  const [status, changeStatus] = useState({
```
Comment (daquinons) [inline src/components/pages/Login/Login.js]: I think it makes sense to put together both states in one object, good one :+1:

File: src/components/pages/Login/LoginForm.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          <Image src={logo} centered alt="tiemendo logo" size="small" />
+
+          <Header as="h3" style={{ marginBottom: '30px' }}>
+            Log in to your account
+          </Header>
+
+          <Form size="large" onSubmit={handleSubmit}>
+            <Segment width={5}>
+              <Form.Input
+                fluid
```
Comment (daquinons) [inline src/components/pages/Login/LoginForm.js]: Is this what is making it responsive? Well done 👍

File: src/components/pages/Login/Login.js (hunk 1 above)
Comment (daquinons) [reply src/components/pages/Login/Login.js]: Oh, I see this PR is only about the responsive. Forget it, I'll put it on Pavel's!

**Comment by Pav0l** [comment]
@abidex4yemi Yemi, disregard Davids review, he ment to review my PR #29 :D

**Pull request merged**